diff/print.go

```diff
@@ -58,11 +56,15 @@ func PrintFileDiff(d *FileDiff) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-func timePtr(ts *pbtypes.Timestamp) *time.Time {
+func timePtr(ts []byte) *time.Time {
 	if ts == nil {
 		return nil
 	}
-	t := ts.Time()
+	var t time.Time
+	err := t.UnmarshalBinary(ts)
+	if err != nil {
```

Comment:
I'd merge those two lines:

```suggestion
	if t.UnmarshalBinary(ts) != nil {
```

or

```
if err := t.UnmarshalBinary(ts); err != nil {
```
depending on how the following comment about ignoring the error will be resolved.